bower.json

```diff
@@ -51,11 +51,15 @@
   },
   "appPath": "app",
   "resolutions": {
-    "lodash": "3.10.1",
     "angular-bootstrap": "0.14.3",
-    "moment": "2.10.6",
-    "matchHeight": "0.6.0",
-    "font-awesome": "4.6.1"
+    "font-awesome": "4.6.3",
+    "patternfly": "3.10.0",
+    "moment": "~2.14.1",
+    "bootstrap-datepicker": "1.6.4",
+    "bootstrap-select": "1.10.0",
+    "bootstrap-touchspin": "~3.1.1",
+    "datatables-colreorder": "1.3.2",
+    "d3": "3.5.17"
```

Comment:
Yeah.  I was getting all sorts of conflicts when I updated.  I believe this is because PatternFly and Angular-PatternFly updated their dependencies.  I could only get resolution on these dependencies by including the resolutions stanza as included.  Am I missing something?